tests/test_custodia.py

```diff
@@ -32,7 +32,7 @@ def find_port(host='localhost'):
 [global]
 server_version = "Secret/0.0.7"
 server_url = ${SOCKET_URL}
-auditlog = test_audit.log
+auditlog = tests/tmp/test_audit.log
```

Comment:
I prefer to pass it in, you are changing it now, and I can see the need to change it again in future, a single variable is easier to change, please introduce a testdir variable.